gazelle/python/file_parser_test.go

```diff
@@ -0,0 +1,259 @@
+// Copyright 2023 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package python
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseImportStatements(t *testing.T) {
+	t.Parallel()
+	type unit struct {
+		name     string
+		code     string
+		filepath string
+		result   []module
+	}
+	units := []unit{
```

Comment:
```suggestion
	units := []struct{
		name     string
		code     string
		filepath string
		result   []module
	}
```

Trying to follow the UBER style guide for go would condense this line a little: https://github.com/uber-go/guide/blob/master/style.md#test-tables